understanding/focus-not-obscured-enhanced.html

```diff
@@ -56,150 +56,109 @@
                </p>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>利点</h2>
                <ul>
                   
-                  <li>Sighted users who rely on a keyboard interface to operate the page will be able to
-                     see the component which gets keyboard focus. Such users include those who rely on
-                     devices which use the keyboard interface, including speech input, sip-and-puff software,
-                     on-screen keyboards, scanning software, and a variety of assistive technologies and
-                     alternate keyboards.
+                  <li>キーボードインタフェースに依存してページを操作している目の見える利用者は、キーボードフォーカスを取得するコンポーネントを見ることができる。そのような利用者には、キーボードインタフェースを利用するデバイス (音声入力、呼気・吸気ソフトウェア、オンスクリーンキーボード、スキャンソフトウェア、並びにさまざまな支援技術及び代替キーボードなど) に依存するような利用者が含まれる。
                   </li>
                   
-                  <li>People with limited or low vision but who rely upon a pointing device (for viewport
-                     orientation and repositioning) benefit from a clearly visible indication of the current
-                     point of keyboard interaction, especially where magnification reduces the overall
-                     useable portion of content.
+                  <li>視力に制限がある又はロービジョンだが、ポインティングデバイス (ビューポートの方向及び位置を変えるため) に依存している利用者は、特に拡大によってコンテンツの全体的な使用可能部分が減少する場合に、キーボード操作の現在位置が明確に表示されることで恩恵を受ける。
                   </li>
                   
-                  <li>People with attention limitations, short term memory limitations, or limitations in
-                     executive processes benefit by being able to more easily discover where the focus
-                     is located.
+                  <li>注意力の制限、短期記憶の制限、又は実行プロセスの制限がある人々は、フォーカスがどこにあるのかをより容易に発見できることで恩恵を受ける。
                   </li>
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>A page has a sticky footer (attached to the bottom of the viewport). When tabbing
-                     down the page the focused item is not at all obscured by the footer because content
-                     in the viewport scrolls up to always display the item with keyboard focus using <a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding" rel="nofollow">scroll padding</a>.
+                  <li>ページにスティッキーフッター (ビューポートの下部に固定されている) がある。ページをタブで下に移動すると、フォーカスされた項目はフッターによって全て隠れることはない。これは、ビューポート内のコンテンツが上にスクロールして、<a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding" rel="nofollow">scroll-padding</a> を使用してキーボードフォーカスのある項目が常に表示されるためである。
```

Comment:
原文では「is not at all obscured 」ですが、「一部たりとも隠されない」というニュアンスを出したいので、以下、いかがでしょうか？
```suggestion
                  <li>ページにスティッキーフッター (ビューポートの下部に固定されている) がある。ページをタブで下に移動すると、フォーカスされた項目はフッターによって少しも隠れることはない。これは、ビューポート内のコンテンツが上にスクロールして、<a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding" rel="nofollow">scroll-padding</a> を使用してキーボードフォーカスのある項目が常に表示されるためである。
```